src/pages/Users/components/ListUsers/ListUsers.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const src = 'https://jsonplaceholder.typicode.com/users';
+
+export const ListUsers = () => {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    axios.get(src).then(data => {
+      setItems(data.data);
+    });
+  }, []);
```

Comment:
create modules for api service, and for query service. 
use async/await

Reply:
![image](https://user-images.githubusercontent.com/43251210/205082742-40a26af5-216e-47f7-90e3-a27f5e39a54a.png)